unit/api/authorization_request_resource.py

```diff
@@ -44,3 +44,11 @@ def decline(self, request: DeclineAuthorizationRequest) -> Union[UnitResponse[Pu
         else:
             return UnitError.from_json_api(response.json())
 
+    def sb_simulate(self, request: SimulateAuthorizationRequest) -> Union[UnitResponse[PurchaseAuthorizationRequestDTO], UnitError]:
```

Comment:
Nit: it might be pretty easy to confuse `sb_*` for just `*` when reading code in our application codebase (potentially from a search results page, etc.) where we use the sandbox calls

I would suggest we change it to `sandbox_*` so it's super clear it's a sandbox call

Reply:
good call!